staging/prometheus-operator/Chart.yaml

```diff
@@ -12,7 +12,7 @@ sources:
   - https://github.com/coreos/kube-prometheus
   - https://github.com/coreos/prometheus-operator
   - https://coreos.com/operators/prometheus
-version: 8.13.16
```

Comment:
not sure i would say this is a patch version anymore based on the changes to the values file. However since we are probably keeping in step with upstream I wont block.